src/BoundaryConditions/field_boundary_conditions.jl

```diff
@@ -32,6 +32,20 @@ mutable struct FieldBoundaryConditions{W, E, S, N, B, T, I}
     immersed :: I
 end
 
+function FieldBoundaryConditions(indices::Tuple, west, east, south, north, bottom, top, immersed)
+    # Turn bcs in windowed dimensions into nothing
+    west, east   = window_boundary_conditions(indices[1], west, east)
+    south, north = window_boundary_conditions(indices[2], south, north)
+    bottom, top  = window_boundary_conditions(indices[3], bottom, top)
+    return FieldBoundaryConditions(west, east, south, north, bottom, top, immersed)
+end
+
```

Comment:
with something like this we can support also different bc for the same field (i.e., open on half of the domain and bounded in the other half). 

It will be useful whenever we can use FFTs (otherwise we can just put an immersed boundary)